app/vmui/packages/vmui/src/components/Views/TableView/TableView.tsx

```diff
@@ -48,6 +48,7 @@ const TableView: FC<GraphViewProps> = ({ data, displayColumns }) => {
         : (d.metric[c.key] || "-")
       )),
       value: d.value ? d.value[1] : "-",
+      values: d.values ? d.values.map(([time, val]) => `${val} @${time}`) : null,
```

Comment:
I think it is better to return an empty array instead of null

Reply:
updated